velox/functions/prestosql/window/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+if(${VELOX_BUILD_TESTING})
+  add_subdirectory(tests)
+endif()
+
+add_library(velox_window OBJECT NthValue.cpp Rank.cpp RowNumber.cpp
+                                WindowFunctionsRegistration.cpp)
+
+target_link_libraries(velox_window velox_exec ${FOLLY_WITH_DEPENDENCIES})
```

Comment:
adding `velox_common_base` should fix the xsimd linux-build-options build issue.